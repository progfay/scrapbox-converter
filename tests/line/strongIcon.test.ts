/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('strongIcon', () => {
  it('Simple strongIcon', () => {
    const input = `Strong Icon
[[progfay.icon]]`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })

  it('Root strongIcon', () => {
    const input = `Strong Icon
[[/icons/hr.icon]]`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
