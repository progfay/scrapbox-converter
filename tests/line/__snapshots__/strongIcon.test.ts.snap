// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`strongIcon Root strongIcon 1`] = `
"<article>
  <h1 class=\\"title\\">Strong&nbsp;Icon</h1>
  <section>
    <p><img src=\\"https://scrapbox.io/api/pages/icons/hr/icon\\" alt=\\"icon\\" class=\\"strong-icon\\"></img></p>
  </section>
</article>"
`;

exports[`strongIcon Simple strongIcon 1`] = `
"<article>
  <h1 class=\\"title\\">Strong&nbsp;Icon</h1>
  <section>
    <p><img src=\\"https://scrapbox.io/api/pages/projectName/progfay/icon\\" alt=\\"icon\\" class=\\"strong-icon\\"></img></p>
  </section>
</article>"
`;
